File: Solved/HandsFree Ha/HandsFreeHa.java
import java.util.Scanner;

/*
Question title = handsfree ha - هندزفری‌ها
https://quera.ir/problemset/110014/
 */

public class HandsFreeHa {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] line1 = sc.nextLine().trim().split("\\W");
        String[] line2 = sc.nextLine().trim().split("\\W");

        String l1 = line1[0];
        String r1 = line1[1];
        String l2 = line2[0];
        String r2 = line2[1];

        boolean result = false;
        if (l1.equals(r2) || l2.equals(r1) || l1.equals(r1) || l2.equals(r2)) {
            result = true;
        }
        if (result)
            System.out.println("YES");
        else
            System.out.println("NO");
    }
}

File: Solved/HandsFree Ha/HandsFree Ha.py
# هندزفری ها
# ID : 110014
# https://quera.org/problemset/110014/
# https://b2n.ir/u60016

line1 = [i for i in input().split(" ")]
line2 = [i for i in input().split(" ")]

l1, r1 = line1
l2, r2 = line2

result = False
if l1 == r2 or l2 == r1 or l1 == r1 or l2 == r2:
    result = True

if result:
    print("YES")

else:
    print("NO")
